src/Postgres/src/Eventuous.Postgresql/Schema.cs

```diff
@@ -15,34 +15,34 @@ public class Schema {
     public Schema(string schema = DefaultSchema) => _schema = schema;
 
     public string StreamMessage      => $"{_schema}.stream_message";
-    public string AppendEvents       => $"{_schema}.append_events";
-    public string ReadStreamForwards => $"{_schema}.read_stream_forwards";
-    public string ReadStreamSub      => $"{_schema}.read_stream_sub";
-    public string ReadAllForwards    => $"{_schema}.read_all_forwards";
-    public string CheckStream        => $"{_schema}.check_stream";
+    public string AppendEvents       => $"select * from {_schema}.append_events(@_stream_name, @_expected_version, @_created, @_messages)";
```

Comment:
What is the reason you changes these lines from SP to SQL statements?

Reply:
@LockTar They aren't stored procedures, but functions. What's the reason it caught your eye?

Reply:
Well it's more that I saw your PR with extension methods for registering the db connection. I'm now busy with a SQL Server sample application and I thought that part of registering connections and stores is different then the EventStoreDb sample app. 

I then saw those changes (not super familiar with PostgreSQL) and I wanted to know why that is. 
Was it not working with before?

Sorry a bit offtopic...

Reply:
Npgsql changed the way stored procedures works in v7, so in <v7 you could use `CommandType.StoredProcedure` to execute functions, but in v7 that is no longer possible hence why I changed them to `CommandType.Text` as per the documentation.

And no worries about the "offtopic" - that's what reviewing code is for, asking questions :)

Reply:
Thank you for the explanation